packages/xpath/test/xforms/jr-itext.test.ts

```diff
@@ -240,17 +240,38 @@ describe('jr:itext(id)', () => {
 			{ expression: 'jr:itext("three")', language: 'en', expected: 'Three' },
 			{ expression: 'jr:itext("one")', language: 'fr', expected: 'Un' },
 			{ expression: 'jr:itext("two")', language: 'fr', expected: 'Deux' },
-			{ expression: 'jr:itext("three")', language: 'fr', expected: 'Trois' },
+			{ expression: 'jr:itext("three")', language: 'fr', expected: 'Ignored' },
```

Comment:
When requesting a `string` value and there are many `value` nodes, XPath returns the text of the first node; in this case, the text is `Ignored`. 

I'm not sure if the expectation is: 
1. Return the text of the first node; then, this is correct.
2. Return the node's text without the `form` attribute; then, we must make some changes to validate each node.

Reply:
Eventually it should return the value without a `form` attribute. I think we could file an issue for that.

Reply:
> it should return the value without a form attribute

This means returning the "default" value of an itext. 

```xml
<itext>
  <translation>
    <value form="<short|image|etc..>"> this is an alternative text/media</value>
    <value> this is the default text </value>
 </translation>
</itext>
```

The expectation in this case is to get: `this is the default text`

Reply:
Exactly, that's the intent for the user-facing `jr:itext` function. But as we've discussed, it's quite rare to use it in form design and if someone does use it in form design they generally won't specify alternative forms.

Reply:
Issue: https://github.com/getodk/web-forms/issues/400 

These test cases are marked as "failing"